### DISCUSSION TITLE
SIWS (Sign In With Solana) with Phantom Ledger Sign

### DISCUSSION THREAD

**Discussion by KishiTheMechanic** [phantom/docs#139] [Help & Support]
### Summary

Our user couldn't use Phantom Ledger to sign in with solana.
https://app.epics.dev/en/

I've asked the user to update Ledger firmware, enable "blind sign", remove and re-add the ledger from the app.

They did the steps but still got errors.

It seems rejected even user tried to sign.
![Screenshot_49](https://github.com/phantom/docs/assets/20677823/75942d88-9640-45fa-9c8c-36b08e2e963b)

Now I'm asking the user about the versions of browser, Ledger and Phantom though, if there is something to try for fixing, please let me know.

Thank you always.
Have a great day!

### Example

https://app.epics.dev/en/

### Steps to Reproduce

Try to sign in with Phantom Ledger

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by KishiTheMechanic** [comment]
SIWS is this.
https://github.com/phantom/sign-in-with-solana

**Reply by adamdelphantom** [reply]
Hi @KishiTheMechanic,

Thank you for posting this! I have reproduced this error. We are looking into the issue. I will follow up once I have more info.

**Reply by KishiTheMechanic** [reply]
Thank you so much🙌

**Comment by KishiTheMechanic** [comment]
Hello! How are you?
Hope you're doing well.

I got a new user to use the Phantom + Ledger with all latest software but can't use Sign In With Solana function.

![2024-01-19_041328](https://github.com/phantom/docs/assets/20677823/399374cf-5665-4dec-8fe3-39b6b1744fd2)

They tried almost everything like these:


I am experiencing issues logging into an environment that uses Phantom and Ledger. 
This issue occurs with my account that holds BDLC and is trying to log in for the first time. 
Although I can log in with a different account on Phantom, I am unable to do so with the account that uses Ledger. 
I have tried various solutions suggested in similar cases, including updating Phantom, Ledger Live, and my browser to their latest versions and enabling blind signing. Despite reinstalling the Phantom wallet, the problem persists.

Initially, I attempted this with a Ledger Nano S Plus as my hardware wallet (HWW), and I also updated the firmware on a previously used Ledger Nano S, but neither approach resolved the issue.
The problem remains the same across different browsers like Brave, Chrome, and Edge, even after installing Phantom on them. When prompted for Ledger authentication, I encounter an error message, and there is no response from the Ledger device when asked for blind signing.


I'm glad if this helps the team to solve this problem.

Have a great day!🙌

Best regards,

**Reply by adamdelphantom** [reply]
Hi @KishiTheMechanic, We have some work to do in order to support SIWS and Ledger. But I don't have an ETA on when that will be delivered. Once I have a sense of when it will be enabled, I will respond here. Thank you for your patience and keeping us updated on your experience!

**Reply by KishiTheMechanic** [reply]
Thank you very much for your update on the SIWS and Ledger integration. I understand that these things take time and appreciate the effort your team is putting into this. It's great to know that the issue is being addressed, and I'm looking forward to the upcoming updates. Please keep up the fantastic work, and know that your efforts are greatly appreciated by the community. We're all excited to see the enhancements you're bringing to the platform!

Best regards,
